### ISSUE TITLE
Compiling 3.7.0 fails on macOS: use of undeclared identifier 'DNF_CONF_COMMANDLINE'

### ISSUE THREAD

**Issue by rb2k** [rpm-software-management/microdnf#102]
I just tried to update the [nix derivation for microdnf](https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/package-management/microdnf/default.nix) from 3.6.0 to 3.7.0, but I run into issues 

```
/tmp/nix-build-microdnf-3.7.0.drv-0/source/dnf/dnf-main.c:591:49: error: use of undeclared identifier 'DNF_CONF_COMMANDLINE'
          dnf_conf_main_set_option ("assumeno", DNF_CONF_COMMANDLINE, "1", NULL);
```

I am a bit confused in that I do indeed not understand where that's supposed to come from.
That being said, my C is also not the best, so maybe I'm missing something obvious.

**Comment by rb2k** [comment]
Ohhhh, new libdnf required?

**Comment by rb2k** [comment]
Actually, @Conan-Kudo : Any chance this line needs to be bumped?

https://github.com/rpm-software-management/microdnf/blob/master/meson.build#L35

```
libdnf = dependency('libdnf', version : '>=0.55.2')
```

Pretty sure 3.7.0 requires an updated libdnf?

**Comment by Conan-Kudo** [comment]
Yeah, I missed that API usage when I was reviewing for 3.7.0 release. You'll need at least libdnf 0.56.0. I'll cut 3.7.1 with this fixed.

**Comment by Conan-Kudo** [comment]
This is fixed now with 9d16cfb6f08a736c5019336738a052bcc99df52f, thanks for the report!

**Comment by Conan-Kudo** [comment]
@rb2k Version 3.7.1 has been released with the fix.
